gopher/enrichment.py

```diff
@@ -22,6 +22,8 @@ def test_enrichment(
     contaminants_filter=None,
     fetch=False,
     progress=False,
+    annotations=pd.DataFrame(),
```

Comment:
**todo:** Change the default for annotations to `None` and in the body add the logic to initialize an empty DataFrame when annotations is `None`. This is the best practice when mutable types are used as default inputs.

The reason is that function arguments are initialized at import time, so the empty DataFrame here would be constructed during the import. However, if subsequent calls to this function modify the DataFrame, then the default value would change between function calls, creating unintended consequences.

Reply:
Thanks for catching that!